pandas/tests/frame/test_arithmetic.py

```diff
@@ -1837,3 +1837,10 @@ def test_arithemetic_multiindex_align():
     )
     result = df1 - df2
     tm.assert_frame_equal(result, expected)
+
+
+def test_bool_frame_mult_float():
+    # GH 18549
+    result = DataFrame(True, list("ab"), list("cd")) * 1.0
```

Comment:
nit, I think the result assignment should be just the op under test

```suggestion
    df = DataFrame(True, list("ab"), list("cd"))
    result = df * 1.0
```